src/main/java/frc/robot/subsystems/BallPath/BallPathImpl.java

```diff
@@ -13,7 +18,32 @@
 // import frc.robot.subsystems.BallPath.Shooter.Shooter;
 // import frc.robot.subsystems.BallPath.Shooter.ShooterImpl;
 
-public class BallPathImpl {
-    
+public class BallPathImpl extends RepeatingPooledSubsystem implements BallPath {
+    public BallPathImpl() {
+        super(10, TimeUnit.MILLISECONDS);
+        // TODO Auto-generated constructor stub
+    }
+
+    // Declare interface with team
+        public void extendOuter(){};
```

Comment:
It's good practice to put an `@Override` annotation on any method that is intended to override an implementation from a superclass, or any method that is declared by an abstract base class or interface. This is a good clue for readers but it also allows the compiler to assist you in making sure that your implementation really is following the declaration it's supposed to, since putting `@Annotation` on a method that is *not* declared by any superclass or interface is a compile-time error (build will fail).